README.md

```diff
@@ -1,278 +1,521 @@
 ```
 
-### Query vectors
+For example, to query by vector values you would pass options like these:
 
 ```ts
-const vector = [...] // a vector
-
-const queryRequest: QueryRequest = {
-  topK: 1,
-  vector,
-  namespace,
-  includeMetadata: true,
-  includeValues: true,
+> await pinecone.index('my-index').query({ topK: 3, vector: [ 0.22, 0.66 ]})
+{
+  matches: [
+    {
+      id: '556',
+      score: 1.00000012,
+      values: [],
+      sparseValues: undefined,
+      metadata: undefined
+    },
+    {
+      id: '137',
+      score: 1.00000012,
+      values: [],
+      sparseValues: undefined,
+      metadata: undefined
+    },
+    {
+      id: '129',
+      score: 1.00000012,
+      values: [],
+      sparseValues: undefined,
+      metadata: undefined
+    }
+  ],
+  namespace: ''
 }
 ```
 
-To query with a sparse vector:
+You include options to `includeMetadata: true` or `includeValues: true` if you need this information. By default these are not returned to keep the response payload small.
+
+Remember that data operations take place within the context of a namespace, so if you are working with namespaces and do not see expected results you should check that you are targeting the correct namespace with your query.
 
 ```ts
-const queryRequest: QueryRequest = {
-  topK: 1,
-  vector,
-  namespace,
-  includeMetadata: true,
-  includeValues: true,
-  sparseVector: {
-    indices: [15, 30, 11],
-    values: [0.1, 0.2, 0.3],
-  },
-};
+const results = await pinecone
+  .index('my-index')
+  .namespace('my-namespace')
+  .query({ topK: 3, vector: [0.22, 0.66] });
 ```
 
-To execute the query:
+#### Querying by vector id
+
+You can query using an existing vector in the index by passing an id.
 
 ```ts
-const queryResponse = await index.query({ queryRequest });
+await pinecone.index('my-index').query({ topK: 10, id: '1' });
 ```
 
-### Update a vector
+#### Hybrid search with sparseVector
+
+If you are working with [sparse-dense vectors](https://docs.pinecone.io/docs/hybrid-search#creating-sparse-vector-embeddings), you can add sparse vector values to perform a hybrid search.
 
 ```ts
-const updateRequest: UpdateRequest = {
-  id: vectorId, // the ID of the vector to update
-  values: [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0], // the new vector values
-  sparseValues: {
-    indices: [15, 30, 11],
-    values: [0.1, 0.2, 0.3],
-  }, // optional sparse values
-  metadata: metadata, // the new metadata
-  namespace,
-};
-await index.update({ updateRequest });
+const pinecone = new Pinecone()
+
+await pinecone.createIndex({
+  name: 'hyrbid-image-search',
+  metric: 'dotproduct',
+  dimension: 512,
+  podType: 's1',
+  waitUntilReady: true
+});
+const index = pinecone.index('hybrid-image-search');
+
+// Create some vector embeddings using your model of choice.
+const vectors = [...]
+
+// Upsert data. Specify a chunkSize if you are trying to upsert
+// a large number of vectors at once
+await index.upsert({ vectors, chunkSize: 100 })
```

Comment:
is chunksize the name of this parameter? seems like it'll get confused with a different concept in NLP. We us batchsize in python

Reply:
Yeah, that's the param name. We took the name from an old util function that Roie created. I'll tell Austin to rename it.

Reply:
Updated in PR: https://github.com/pinecone-io/pinecone-ts-client/pull/90

thanks for the callout